src/partitions.jl

```diff
@@ -104,7 +106,7 @@ function nextfixedpartition(n, m, bs)
     as = copy(bs)
     if isempty(as)
         # First iteration
-        as = [n-m+1; ones(Int, m-1)]
+        as = [[n-m+1]; ones(Int, m-1)]
```

Comment:
`(n-m+1):(n-m+1)` and maybe mark this as workaround.

`as = ones(Int, m); as[1] = n - m + 1` will likely be faster than any of these...